gateway/src/main/java/org/georchestra/gateway/autoconfigure/app/FiltersAutoConfiguration.java

```diff
@@ -65,4 +69,9 @@ public class FiltersAutoConfiguration {
         return new StripBasePathGatewayFilterFactory();
     }
 
+    @Bean
+    RouterFunction<ServerResponse> staticResourceRouter() {
+        return RouterFunctions.resources("/**", new ClassPathResource("static/"));
+    }
+
```

Comment:
what is the purpose of introducing this bean ?

Reply:
static folder which contains css files is not accessible by default on gateway. We need this bean to make it accessible when redirection is set to gateway by Apache proxy.

Reply:
so it's more of a bugfix unrelated to the current PR, ok, I was not aware we already had a static folder in the classpath.

Reply:
static folder is accessible if no Apache redirection is set, i think this bug is regression when using Apache Proxy.

Reply:
> static folder is accessible if no Apache redirection is set, i think this bug is regression when using Apache Proxy.


I think this is specific to the private docker composition (apache / traefik config) that I wrote to make the development easier (https://github.com/camptocamp/inrae-renater-docker/blob/master/docker-compose.yaml#L65-L82) ; if we already have a /static route in the gateway and it works, then we don't probably need the bean to be defined, but we could check this when deploying an early modified version of the gateway.